src/emuseum.js

```diff
@@ -54,6 +73,15 @@ module.exports = ({ emuseumKey }) => {
         })
       )
       return response
+    },
+    async getRawObjectsAll () {
+      if (!emuseumKey) throw MISSING_EMUSEUM_KEY
+
+      const uri = `http://egallery.williams.edu/objects/json?key=${emuseumKey}`
```

Comment:
confirmed this gives all object? unpaginated? thats pretty damn nice

I wonder if the performance is good enough if we just delegate to grabbing all, and we use in memory cache to search ourselves after the first hit for queries with ids. either way, thats for another PR.

Reply:
It returns just a first page I believe. And I did a quick test of pagination which the api does support. So I'll add a todo to make this properly paginated